screens/ProfileScreen/Navigation/ProfileScreensNavigation.tsx

```diff
@@ -0,0 +1,42 @@
+import { StackScreenProps } from "@react-navigation/stack"
+import { ProfileScreenProps } from "../ProfileScreen"
+import { StackNavigatorType } from "@components/Navigation";
+import { SettingsScreen } from "@screens/SettingsScreen/SettingsScreen";
+import { ProfileScreenNavWrapper } from "./ProfileScreenNavWrapper";
+import { EditProfileScreenNavWrapper } from "./EditProfileScreenNavWrapper";
+
+
+export type ProfileScreensParamsList = {
+  "Profile Screen": ProfileScreenProps
+  "Edit Profile Screen": ProfileScreenProps
+  "Settings Screen": undefined
```

Comment:
Stick with camel case names. We've already done it with the reporting screens, and it makes this type look more like a standard typescript type.